### PR TITLE
New pre-release: v4.7.0-rc1

### PR DESCRIPTION
**Pull request by lsm5**
#### Does this PR introduce a user-facing change?

```release-note
None
```

Let's get https://github.com/containers/podman/pull/19993 merged first.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,109 @@
...
+- The `podman push` and `podman manifest push` commands now support the `--force-compression` optionto prevent reusing other blobs ([#18860](https://github.com/containers/podman/issues/18660)).
+- The `podman manifest push` command now supports `--add-compression` to push with compressed variants.
+- The `podman manifest push` command now honors the `add_compression` field from containers.conf if `--add-compression` is not set.
+- The `podman run` and `podman create --mount` commands now support the `ramfs` type ([#19659](https://github.com/containers/podman/issues/19659)).
+- When running under systemd (e.g., via Quadlet), Podman will extend the start timeout in 30 second steps up to a maximum of 5 minutes when pulling an image.
+- The `--add-host` option now accepts the special string `host-gateway` instead of an IP Address, which will be mapped to the host IP address.
+- The `podman generate systemd` command is deprecated.  Use Quadlet for running containers and pods under systemd.
+- The `podman secret rm` command now supports an `--ignore` option.
+- The `--env-file` option now supports multiline ([#18724](https://github.com/containers/podman/issues/18724)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:34]: multiline to multiline variables

Comment (lsm5) [reply RELEASE_NOTES.md:34]: hmm, I'll fix this in the published release. But hope we can skip this here.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,109 @@
...
+- The `--read-only-tmpfs` flag now affects /dev and /dev/shm as well as /run, /tmp, /var/tmp ([#12937](https://github.com/containers/podman/issues/12937)).
+- The Podman --mount option now supports bind mounts passed as globs.
+- The `--mounts` option can now be specified in containers.conf.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:37]: is it "mount" or "mounts"?  I see one in line 36 and the other in line 37

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,109 @@
...
+- Temporary files created when dealing with images in `/var/tmp` will automatically be cleaned up on reboot.
+- There is now a new filter option `since` for `podman volume ls` and `podman volume prune` ([#19228](https://github.com/containers/podman/issues/19228)).
+- The `podman inspect` command now has tab-completion support ([#18672])(https://github.com/containers/podman/issues/18672)).
+- The `podman kube play` command now has support for the use of reserved annotations in the generated YAML.
+- The progress bar is now displayed when decompressing a Podman machine image ([#19240](https://github.com/containers/podman/issues/19240)).
+- The `podman secret inspect` command supports a new option `--showsecret` which will output the actual secret.
+- The `podman secret create` now supports a `--replace` option, which allows you to modify secrets without replacing containers.
+- The `podman login` command can now read the secret for a registry from its secret database created with podman secret create ([#18667]](https://github.com/containers/podman/issues/18667)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:47]: back ticks for `podman secret create`

File: RELEASE_NOTES.md (hunk 2 above)
Comment (lsm5) [reply RELEASE_NOTES.md:37]: It's mounts in containers.conf. I can fix in published release. Wonder if I should resurrect the release notes PR.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (lsm5) [reply RELEASE_NOTES.md:47]: will fix in release.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,109 @@
...
+- Quadlet now recursively calls in user directories for unit files.
+- Quadlet now allows the user to set the service working directory relative to the YAML or Unit files ([17177](https://github.com/containers/podman/discussions/17177)).
+- Quadlet now allows setting user-defined names for `Volume` and `Network` units via the `VolumeName` and `NetworkName` directives, respectively.
+- Kube quadlets can now support autoupdate.
+
+### Bugfixes
+- Fixed an issue where containers were being restarted after a `podman kill`.
+- Fixed a bug where events could report incorrect healthcheck results ([#19237](https://github.com/containers/podman/issues/19237).
+- Fixed a bug where running a container in a pod didn't fail if volumes or mounts were specified in the containers.conf file.
+- Fixed a bug where pod cgroup limits where not being honored after a reboot ([#19175](https://github.com/containers/podman/issues/19175)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:74]: "where not" to "were not"

**Comment by TomSweeneyRedHat** [comment]
LGTM
A couple of nits in the docs, for the interest of time, I'm fine with merging now and touching up later.

**Comment by TomSweeneyRedHat** [comment]
Do we know why/how 34 tests were canceled?

**Comment by lsm5** [comment]
> Do we know why/how 34 tests were canceled?

@TomSweeneyRedHat they show up as canceled in UI and stay that way until they succeed I guess. Happened yesterday on buildah too. I retriggered those tests and they were running. But anyway, I've pushed again with the fixes you mentioned so things should run now.

**Comment by lsm5** [comment]
ugh, the RELEASE NOTES still needs a fix. The option is called `--mount` but containers.conf says mounts.

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,109 @@
...
+- The `podman run` and `podman create --mount` commands now support the `ramfs` type ([#19659](https://github.com/containers/podman/issues/19659)).
+- When running under systemd (e.g., via Quadlet), Podman will extend the start timeout in 30 second steps up to a maximum of 5 minutes when pulling an image.
+- The `--add-host` option now accepts the special string `host-gateway` instead of an IP Address, which will be mapped to the host IP address.
+- The `podman generate systemd` command is deprecated.  Use Quadlet for running containers and pods under systemd.
+- The `podman secret rm` command now supports an `--ignore` option.
+- The `--env-file` option now supports multiline variables ([#18724](https://github.com/containers/podman/issues/18724)).
+- The `--read-only-tmpfs` flag now affects /dev and /dev/shm as well as /run, /tmp, /var/tmp ([#12937](https://github.com/containers/podman/issues/12937)).
+- The Podman `--mount` option now supports bind mounts passed as globs.
+- The `--mount` option can now be specified in containers.conf using the `mounts` field.
```
Comment (lsm5) [inline RELEASE_NOTES.md:37]: @TomSweeneyRedHat PTAL

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -1,5 +1,109 @@
...
+- Quadlet now recursively calls in user directories for unit files.
+- Quadlet now allows the user to set the service working directory relative to the YAML or Unit files ([17177](https://github.com/containers/podman/discussions/17177)).
+- Quadlet now allows setting user-defined names for `Volume` and `Network` units via the `VolumeName` and `NetworkName` directives, respectively.
+- Kube quadlets can now support autoupdate.
+
+### Bugfixes
+- Fixed an issue where containers were being restarted after a `podman kill`.
+- Fixed a bug where events could report incorrect healthcheck results ([#19237](https://github.com/containers/podman/issues/19237).
+- Fixed a bug where running a container in a pod didn't fail if volumes or mounts were specified in the containers.conf file.
+- Fixed a bug where pod cgroup limits where not being honored after a reboot ([#19175](https://github.com/containers/podman/issues/19175)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:74]: looks like you fixed where not below, but not here where I picked it up?

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**